src/pages/docs/creators/python/tutorials/building-custom-image.md

```diff
@@ -0,0 +1,174 @@
+
+```dockerfile
+FROM node:latest
+WORKDIR /golem/work
+VOLUME /golem/work
+COPY Dockerfile /golem/info/description.txt
+COPY Dockerfile /golem/work/info.txt
+```
+
+Note we copy Dockerfile content into 2 different locations:
+
+- to /golem/info (this folder is not defined as VOLUME)
+- and to /golem/work (this folder is defied as VOLUME)
+
+## Building the Docker image
+
+To build the Docker image from the `Dockerfile`, we can run the following command in the same directory (`.`) as the Dockerfile to build an image tagged `golem-node`:
+
+{% tabs %}
+{% tab label="Linux" %}
+
+```bash
+docker build -t golem-node .
+```
+
+{% /tab %}
+{% tab label="macOS" %}
+
+```bash
+docker build --platform linux/amd64 -t golem-node .
+```
+
+{% /tab %}
+{% tab label="Windows" %}
+
+```bash
+docker build -t golem-node .
+```
+
+{% /tab %}
+{% /tabs %}
+
+The output should look like this:
+
+![](/image_tutorial_build.png)
+
+{% alert level="info" %}
+
+Note that the image won't be turned into a file in the same directory. The location of the actual Docker image file depends on the Docker and your operating system version. Please consult the Docker manual for additional information.
+
+{% /alert  %}
+
+## Converting from Docker to Golem and uploading it to the registry
+
+Now when you have a Docker image built, we can convert it to a Golem image. To save time, we will also upload it to the registry with the same command. To do this, you need to run the appropriate command that uses `gvmkit-build` to convert and push the image `golem-node` to the registry.
+
+{% alert level="info" %}
+
+If you do not have `gvmkit-build` installed please follow [installation intructions](/docs/creators/javascript/examples/tools/gvmkit-build-installation). You can also use it without installation using `npx` or `pipx` commands.
+
+{% /alert  %}
+
+{% tabs %}
+{% tab label="JavaScript" %}
+
+```bash
+gvmkit-build golem-node --push --nologin
+```
+
+{% /tab %}
+{% tab label="Python" %}
+
+```bash
+gvmkit-build golem-node --push --nologin
+```
+
+{% /tab %}
+{% /tabs %}
+
+After running the command, you will see an output that looks like this:
+
+![](/image_tutorial_upload.png)
+
+The hash is found after the `image link`, which in this case gives us the hash `8b238595299444d0733b41095f27fadd819a71d29002b614c665b27c`. If you ever lose your hash, you can always recover/re-generate it by running the same command again.
+
+## Using the image in a requestor script
+
+Let's use the newly created image in a requestor script and ensure this is the one we have just made.
+
+We need to prepare our environment:
+
+```bash
+mkdir golem-example
+cd golem-example
+npm init
+npm i @golem-sdk/golem-js
+```
+
+We can now create our `index.mjs` requestor file, with the `package: "8b238595..."` matching our image hash.
```

Comment:
ugh... this is a Python tutorial... and the instructions pertain to JS...

Reply:
true, need to add a Python snippet here, fro now will indicate the tutorial is JS